spec/setup_test_database.rb

```diff
@@ -0,0 +1,6 @@
+require 'pg'
+
+def setup_test_database
+  connection = PG.connect(dbname: 'chitter_test')
```

Comment:
this could utilise your DatabaseConnection class rather than manually connecting to PG again